contents/ios/IntroToIos.md

```diff
@@ -0,0 +1,105 @@
+
+Compatibility issues, especially with UI components, can come up when you are trying to develop a mobile app using cross-platform solutions, usually because some of these components are available on one OS but not the other, hence the cross-platform API will completely not give the component as an option to developers. 
+
+In a nutshell, there are many factors to consider when deciding to develop mobile apps using cross-platform or native solutions, but as budding software engineers, it definitely would not hurt to pick up both!
+
+## 4. Getting started with native iOS development
+
+To get started with native iOS development, we have to first get some of the basic tools set up:
+
+- To use Xcode and write native iOS apps in Swift, you will need MacOS. If you are using a non-Mac operating system (Windows, Linux or others), you can install a MacOS Virtual Machine on your computer. This [blog post](https://medium.com/@twister.mr/installing-macos-to-virtualbox-1fcc5cf22801) is an excellent tutorial on how this can be done. If you already own a Macbook or a Mac desktop, you are all set for this step.
+- Next, you should register for a free Apple Developer Account. Don't confuse this with the paid iOS Developer Program! Anyone can register for the Apple Developer Account for free. Go to [Apple's Developer Website](https://developer.apple.com/register/) to do this. It requires you to have an Apple ID, so you can use your existing one if you already have an Apple ID.
+- Once you have an Apple Developer Account, you can either directly download Xcode from the [website](https://developer.apple.com/xcode/), or search for Xcode on your Mac App Store. Xcode is the Integrated Development Environment (IDE) that provides you with everything you need to develop an iOS app from scratch. It also somes with the iPhone and iPad simulator that you will need to test your application.
+
+Native iOS applications can be written in [Objective-C or Swift](https://android.jlelse.eu/objective-c-or-swift-which-technology-to-learn-for-ios-app-development-3c681d1a05ac). Swift is a relatively newer language, introduced only in 2014, while Objective-C is more of an old school programming language. If you are just starting out on iOS development, you should strongly consider using Swift. The main reason being that many of the documentations and help on the internet are written in Swift and meant for iOS development in Swift, hence it will be easier to look for resources that target Swift instead of Objective-C. Furthermore, Apple mostly regards Swift as the main language for iOS programming now, and Objective-C support is now more of a backward-compatability. 
+
+If you are new to Swift, you may want to read up on our Swift article right on this website: [Introduction to Swift]({{baseUrl}}/contents/swift/welcome-to-swift.html)
```

Comment:
Some articles on optionals, exception handling and memory management (strong-weak references) might be good.

Reply:
I think these are more related to the language Swift itself, rather than iOS mobile development. Some of these topics are already addressed on our own Swift page :)